x/market/types/params.go

```diff
@@ -19,13 +19,37 @@ var (
 	KeyPoolRecoveryPeriod = []byte("PoolRecoveryPeriod")
 	// Min spread
 	KeyMinStabilitySpread = []byte("MinStabilitySpread")
+	KeyMaxSupplyCoin      = []byte("MaxSupplyCoin")
 )
 
 // Default parameter values
 var (
 	DefaultBasePool           = sdk.NewDec(1000000 * core.MicroUnit) // 1000,000sdr = 1000,000,000,000usdr
 	DefaultPoolRecoveryPeriod = core.BlocksPerDay                    // 14,400
-	DefaultMinStabilitySpread = sdk.NewDecWithPrec(2, 2)             // 2%
+	DefaultMinStabilitySpread = sdk.NewDecWithPrec(2, 2)
+	DefaultMaxSupplyCoin      = []sdk.Coin{
+		{Denom: "uluna", Amount: sdk.NewInt(10000000000)},
```

Comment:
I need some help to understand this, isn't uluna a LUNC? for me to limit 10 Billions of LUNC how much uluna do I need to inform? Does this logic work for stablecoins like USTC, EUTC? In the case of stablecoins, I still don't know a value to limit, we need to solve this via governance, because I think that modes other than USTC and LUNC, we need to limit a value greater than this Today to have a margin of exchange. Ex. if EUTC is at 2,000,000 limit it to 200,000,000! But that's not going to be the ones we're going to decide!

Reply:
For max supply of 10billion Luna's it have to be 10,000,000,000,000,000 µLuna

Reply:
Good morning @vegasmorph , I developed it to help the L1 team, I think now they can make the adjustments. You need to carry out studies to limit the other currencies, for me the others should have a margin for the exchange. In the case of LUNC and USTC there will be no margin. The USTC doesn't even know what the ideal limit is, but the other currencies I will take to the governance to approve a maximum value until then I will list all the supply values and inform in the proposal and say how much would be ideal.